### PR TITLE
[C/C++] Adding 2020/Nov release notes

### PR DESCRIPTION
**Pull request by vhvb1989**

### CODE REVIEW

File: releases/2020-11/cpp.md [hunk 1]
```diff
@@ -8,35 +8,139 @@ repository: Azure/azure-sdk-for-cpp
...
-#### Updates
-
-- _Add packages_
 
 #### Beta
 
-- _Add packages_
+- Azure Core
+- Azure Identity
+- Azure Storage Blobs
```
Comment (ahsonkhan) [inline releases/2020-11/cpp.md:16]: Add `Azure Storage Common`?

File: releases/2020-11/cpp.md [hunk 2]
```diff
@@ -8,35 +8,139 @@ repository: Azure/azure-sdk-for-cpp
...
+
+git checkout <tag_name>
 ```
 
 ## Feedback
 
 If you have a bug or feature request for one of the libraries, please post an issue to [GitHub](https://github.com/Azure/azure-sdk-for-cpp/issues).
 
 ## Release highlights
+### azure-identity [Changelog](https://github.com/Azure/azure-sdk-for-cpp/blob/master/sdk/identity/azure-identity/CHANGELOG.md#100-beta1-2020-11-11)
```
Comment (ahsonkhan) [inline releases/2020-11/cpp.md:35]: nit: add new line between heading

Comment (ahsonkhan) [reply releases/2020-11/cpp.md:35]: nit: Move the identity section to after core

File: releases/2020-11/c.md [hunk 3]
```diff
@@ -34,9 +28,25 @@ If you have a bug or feature request for one of the libraries, please post an is
 
 ## Release highlights
 
-### _Package name_
+### azure-sdk-for-c [Changelog](https://github.com/Azure/azure-sdk-for-c/blob/feature/iot_pnp/CHANGELOG.md#110-beta2-2020-11-11)
```
Comment (ahsonkhan) [inline releases/2020-11/c.md:31]: This feature branch will likely be deleted, resulting in this link being a 404 in the future.
We should point to master instead.

I have submitted a fix to the master changelog so it is correct, please use that:
https://github.com/Azure/azure-sdk-for-c/pull/1522

Comment (ahsonkhan) [reply releases/2020-11/c.md:31]:
```suggestion
### azure-sdk-for-c [Changelog](https://github.com/Azure/azure-sdk-for-c/blob/master/CHANGELOG.md#110-beta2-2020-11-11)
```

**Review by ahsonkhan — APPROVED** [review]
Otherwise, looks good. It would be nice to get another review though (maybe @jongio, @RickWinter, @czubair).

File: releases/2020-11/cpp.md [hunk 4]
```diff
@@ -34,9 +34,114 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+  - Signature updated for `SetHost`, `SetPath` and `AppendPath`.
+  - Removed `SetFragment`.
+  - Renamed `AppendQueries` to `AppendQueryParameters`.
+  - Renamed `AppendQuery` to `AppendQueryParameter`.
+  - Renamed `RemoveQuery` to `RemoveQueryParameter`.
+  - Renamed `GetQuery` to `GetQueryParameters`.
+
+#### Bug Fixes
+
+- Prevent pipeline of length zero to be created.
```
Comment (RickWinter) [inline releases/2020-11/cpp.md:68]:
```suggestion
- Prevent pipeline of length zero.
```

File: releases/2020-11/cpp.md [hunk 5]
```diff
@@ -34,9 +34,114 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- Add entropy to `Uuid` generation.
+
+#### Other changes and Improvements
+
+- Add high-level and simplified core.hpp file for simpler include experience for customers.
+- Add code coverage using gcov with gcc.
+- Update SDK-defined exception types to be classes instead of structs.
+- Updated `TransportException` and `InvalidHeaderException` to derive from `RequestFailedException`.
+- Vcpkg dependency version updated to 2020.11.
+- Make libcurl network requests cancelable by Context::Cancel().
```
Comment (RickWinter) [inline releases/2020-11/cpp.md:79]: In the future I'd call this a feature.  Not that big a deal for this change.

File: releases/2020-11/cpp.md [hunk 6]
```diff
@@ -34,9 +34,114 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- Validate HTTP headers for invalid characters.
+- Calling `Cancel()` from context now throws `OperationCanceledException`.
```
Comment (RickWinter) [inline releases/2020-11/cpp.md:81]: This is a breaking change

File: releases/2020-11/cpp.md [hunk 7]
```diff
@@ -34,9 +34,114 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- Rename header file `storage_error.hpp` to `storage_exception.hpp`.
+- Rename `SharedKeyCredential::SetAccountKey` to `SharedKeyCredential::UpdateAccountKey`.
+- Rename `AccountSasBuilder::ToSasQueryParameters` to `AccountSasBuilder::GenerateSasToken`.
+- Remove `storage_version.hpp` and add `version.hpp`.
+- Make `SharedKeyCredential` a class.
+
+#### Other Changes and Improvements
+
+- Remove support for specifying SAS version.
+
```
Comment (RickWinter) [inline releases/2020-11/cpp.md:105]:
```suggestion
```

**Review by RickWinter — APPROVED** [review]

File: releases/2020-11/cpp.md (hunk 6 above)
Comment (vhvb1989) [reply releases/2020-11/cpp.md:81]: It was already throwing in the past. We are just changing the exception, that's why it was not considered as a breaking change

File: releases/2020-11/cpp.md (hunk 4 above)
Comment (vhvb1989) [reply releases/2020-11/cpp.md:68]: Can we publish a different change log here than the one that is published directly in the repo?

File: releases/2020-11/cpp.md (hunk 6 above)
Comment (RickWinter) [reply releases/2020-11/cpp.md:81]: That is a breaking change for those that were catching only the specific exception.

File: releases/2020-11/cpp.md (hunk 4 above)
Comment (RickWinter) [reply releases/2020-11/cpp.md:68]: That's a good point.  We can.  However I agree with you its not a good practice so lets leave what was there.

File: releases/2020-11/cpp.md (hunk 6 above)
Comment (ahsonkhan) [reply releases/2020-11/cpp.md:81]: What was the exception being thrown before? Going from a general type (say runtime_error) to a specific one that derives from it, wouldn't really be considered breaking.

**Pull request merged**